src/game/map.rs

```diff
@@ -164,13 +163,12 @@ pub fn find_route_with_callback(
 
     // Type erased and boxed callback: no longer a type specific to the closure
     // passed in, now unified as &Fn
-    let callback_type_erased: &mut (dyn FnMut(RoomName, RoomName) -> f64) =
-        &mut callback_boxed;
+    let callback_type_erased: &mut (dyn FnMut(RoomName, RoomName) -> f64) = &mut callback_boxed;
```

Comment:
Do you have a special `rustfmt` config? Line length should be pretty standard; I don't get why it's jumping around.

Reply:
Nothing custom, I'm just gonna keep blaming nightly builds ;)